### ISSUE TITLE
linking data to publications in DOI metadata

### ISSUE THREAD

**Issue by dnoesgaard** [gbif/content-crawler#21]
I've been in touch with various people at Crossref and Datacite regarding their Event Data services that are supposed to track various "events" relating to publications and data. One example of such events is the use of data in a publication. Ideally, authors using GBIF data would include a properly formatted citation including a download DOI, and this would be enough for the link to be established. However, journal publishers vary in their use of DOI metadata for references, and some don't even use it :(

But - we should be able to help this along by adding relevant _relatedIdentifier_ tags (using _relationType_="IsCitedBy") to DOIs that has been identified in our literature tracking efforts. For example, by manual discovery we've found that [this paper](https://doi.org/10.1111/nph.14534) has made use of these datasets (10.15468/dl.x1u7tg, 10.15468/dl.oygtqo and 10.15468/dl.ynmkwo), so we could add records to the metadata of the three DOIs:
 
`<relatedIdentifier relatedIdentifierType="DOI" relationType="IsCitedBy" >10.1111/nph.14534</relatedIdentifier>`

Obviously, discovery is manual and proper DOI citations are still relatively rare, so this could be done as manually as it makes sense.

Thoughts? @kbraak @mdoering @cgendreau

**Comment by timrobertson100** [comment]
This has been happening but manually.

The remaining task is to automate this as we crawl mendeley

**Comment by mdoering** [comment]
Should these manually curated citations live in the registry so we can recreate the latest metadata from there and have it at hand whenever the DOI metadata needs a change?

**Comment by dnoesgaard** [comment]
I was thinking about @mdoering's point yesterday, and I think it makes a lot of sense to have the full DOI metadata available in one place.

Perhaps the logical step would be to fully integrate the literature index into the registry?

**Comment by timrobertson100** [comment]
Automating the update and the entering and editing of citations are different concerns. We can fully automate this without moving citations into the registry DB / interface, so I suggest we don't conflate these aspects.

@dnoesgaard - can you open an issue on registry that documents what could be of benefit if we moved the citation into internal registry storage? I recall the original reasons to use the external service were 1) promote more outside use (i.e. crowd source this) and 2) minimize tool development at that time. We can revisit the decision but we should clarify what we want to achieve from doing this.

**Comment by dnoesgaard** [comment]
I agree completely. I wasn't referring to entering/editing, but the fact the we store citations in a separate db (ES). I guess my point was that whenever a DOI metadata update is triggered for whatever reason, we should have all metadata available (including citations).

**Comment by timrobertson100** [comment]
Ah, thanks for the clarification - it makes sense.
Updating DOI is one motivation, but another would be easier reporting using SQL for e.g. accumulations by hosting organisation or country etc.

**Comment by dnoesgaard** [comment]
Right. A lot of folks are asking about access to these citations through the official API.
